storm-client/test/py/test_storm_cli.py

```diff
@@ -93,7 +93,7 @@ def test_localconfvalue_command(self):
         self.base_test(
             ["storm", "localconfvalue", "conf_name"], self.mock_popen, mock.call([
              self.java_cmd, '-client', '-Dstorm.options=',
-             '-Dstorm.conf.file=', '-cp',  self.storm_dir + '/*:' + self.storm_dir + '/lib:' + self.storm_dir +'/extlib:' + self.storm_dir + '/extlib-daemon:' + self.storm_dir + '/conf',
+             '-Dstorm.conf.file=', '-cp',  self.storm_dir + '/*:' + self.storm_dir + '/lib/*:' + self.storm_dir +'/extlib:' + self.storm_dir + '/extlib-daemon:' + self.storm_dir + '/conf',
```

Comment:
what determines if a directory needs the * or not?  Why not add it to extlib and/or all directories?

Reply:
All directories with jars should show in classpath with "/*". I will raise a separate Jira to check if storm.py is doing it and then also fix this test_storm_cli.py. 

The test seems to succeed with just this change.